tests/docker/Dockerfile

```diff
@@ -163,6 +163,10 @@ RUN apt update && \
       file && \
     rm -rf /var/lib/apt/lists/*
 
+RUN mkdir /root/external_test_suites && \
+    git -C /root/external_test_suites clone --depth=1 https://github.com/getsentry/arroyo.git && \
```

Comment:
based on the rest of the Dockerfile it looks like we try to pin everything to a specific sha1/version

Reply:
Yeah. It's definitely a trade-off. It's more stable if we pin, but then someone needs to remember to update it manually every now and then. That being said, that test suite doesn't seem to change much over time. I'll pin it to the current tip of the main branch.

Reply:
yeh, agree about updating. i think we need some out-of-band process by which we periodically upgrade the dependencies. at least this way we keep flakiness minimized